### ISSUE TITLE
`/api/v2/exhibits/{id}` route returns `AttributeError`

### ISSUE THREAD

**Issue by foglabs** [WGBH-MLA/ov-wag#107]
### What happened?

When I GET `/api/v2/exhibits/:id` from the frontend, I get this error from ov-wag:

```
ov-wag-wagtail-1  | AttributeError: Got AttributeError when attempting to get a value for field `cover_image` on serializer `ExhibitsOrderableSerializer`.
ov-wag-wagtail-1  | The serializer field might be named incorrectly and not match any attribute or key on the `ExhibitsOrderable` instance
```

### Steps to reproduce

1. send a GET request to /api/v2/exhibits/:id

### Relevant logs

```shell
ov-wag-wagtail-1  | Internal Server Error: /api/v2/exhibits/5/
ov-wag-wagtail-1  | Traceback (most recent call last):
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/fields.py", line 446, in get_attribute
ov-wag-wagtail-1  |     return get_attribute(instance, self.source_attrs)
ov-wag-wagtail-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/fields.py", line 96, in get_attribute
ov-wag-wagtail-1  |     instance = getattr(instance, attr)
ov-wag-wagtail-1  |                ^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/app/exhibits/models.py", line 34, in cover_image
ov-wag-wagtail-1  |     return self.exhibit.cover_image
ov-wag-wagtail-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  | AttributeError: 'NoneType' object has no attribute 'cover_image'
ov-wag-wagtail-1  | 
ov-wag-wagtail-1  | During handling of the above exception, another exception occurred:
ov-wag-wagtail-1  | 
ov-wag-wagtail-1  | Traceback (most recent call last):
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
ov-wag-wagtail-1  |     response = get_response(request)
ov-wag-wagtail-1  |                ^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
ov-wag-wagtail-1  |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
ov-wag-wagtail-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/wagtail/api/v2/router.py", line 68, in wrapped
ov-wag-wagtail-1  |     return func(request, *args, **kwargs)
ov-wag-wagtail-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
ov-wag-wagtail-1  |     return view_func(*args, **kwargs)
ov-wag-wagtail-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/viewsets.py", line 125, in view
ov-wag-wagtail-1  |     return self.dispatch(request, *args, **kwargs)
ov-wag-wagtail-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 509, in dispatch
ov-wag-wagtail-1  |     response = self.handle_exception(exc)
ov-wag-wagtail-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/wagtail/api/v2/views.py", line 132, in handle_exception
ov-wag-wagtail-1  |     return super().handle_exception(exc)
ov-wag-wagtail-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 469, in handle_exception
ov-wag-wagtail-1  |     self.raise_uncaught_exception(exc)
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
ov-wag-wagtail-1  |     raise exc
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 506, in dispatch
ov-wag-wagtail-1  |     response = handler(request, *args, **kwargs)
ov-wag-wagtail-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/wagtail/api/v2/views.py", line 89, in detail_view
ov-wag-wagtail-1  |     return Response(serializer.data)
ov-wag-wagtail-1  |                     ^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/serializers.py", line 555, in data
ov-wag-wagtail-1  |     ret = super().data
ov-wag-wagtail-1  |           ^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/serializers.py", line 253, in data
ov-wag-wagtail-1  |     self._data = self.to_representation(self.instance)
ov-wag-wagtail-1  |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/wagtail/api/v2/serializers.py", line 348, in to_representation
ov-wag-wagtail-1  |     data[field.field_name] = field.to_representation(attribute)
ov-wag-wagtail-1  |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/relations.py", line 565, in to_representation
ov-wag-wagtail-1  |     return [
ov-wag-wagtail-1  |            ^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/relations.py", line 566, in <listcomp>
ov-wag-wagtail-1  |     self.child_relation.to_representation(value)
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/wagtail/api/v2/serializers.py", line 128, in to_representation
ov-wag-wagtail-1  |     return serializer.to_representation(value)
ov-wag-wagtail-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/wagtail/api/v2/serializers.py", line 339, in to_representation
ov-wag-wagtail-1  |     attribute = field.get_attribute(instance)
ov-wag-wagtail-1  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ov-wag-wagtail-1  |   File "/usr/local/lib/python3.11/site-packages/rest_framework/fields.py", line 479, in get_attribute
ov-wag-wagtail-1  |     raise type(exc)(msg)
ov-wag-wagtail-1  | AttributeError: Got AttributeError when attempting to get a value for field `cover_image` on serializer `ExhibitsOrderableSerializer`.
ov-wag-wagtail-1  | The serializer field might be named incorrectly and not match any attribute or key on the `ExhibitsOrderable` instance.
```

**Comment by mrharpo** [comment]
@foglabs I can't reproduce this. Instead, I'm getting a different error:

## Backend
First, when getting the home page, the frontend requests `GET /api/v2/exhibits` which `301`  redirects to `/api/v2/exhibits/`

After that, there is a request to `/exhibit/34`, which `301` redirects to `/api/v2/exhibit/34/`, and then errors with `404`, because the route should be `/api/v2/exhibits/34/`

## Frontend

Meanwhile, the frontend looks like it's trying to `GET /exhibits/34`, but says it's a `500`, 

However, the backend has no such request, and no such `500`
```
duh ex <!DOCTYPE html>
<html class="no-js">
    <head>
        <meta charset="utf-8" />
        <title>Internal server error</title>
        <meta name="viewport" content="width=device-width, initial-scale=1" />
    </head>
    <body>
        <h1>Internal server error</h1>

        <h2>Sorry, there seems to be an error. Please try again soon.</h2>
    </body>
</html>

No images found! Can't build the thumb list without images. If you don't need thumbs, set showThumbs={false} in the Carousel. Note that it's not possible to get images rendered inside custom components. More info at https://github.com/leandrowd/react-responsive-carousel/blob/master/TROUBLESHOOTING.md
GET / 500 - - 178.486 ms
duh ex {
  meta: { total_count: 2 },
  items: [
    {
      id: 34,
      meta: [Object],
      title: 'Exhibit A',
      body: '<h2 data-block-key="e5cyz">Hi!</h2><p data-block-key="desda">It&#x27;s me!</p>',
      cover_image: [Object],
      cover_thumb: [Object],
      hero_image: [Object],
      hero_thumb: [Object],
      authors: [Array]
    },
    {
      id: 35,
      meta: [Object],
      title: 'Exhibit B',
      body: '<h2 data-block-key="rng4q">Another Exhibit!</h2><p data-block-key="2tdfb">With content!</p>',
      cover_image: [Object],
      cover_thumb: [Object],
      hero_image: [Object],
      hero_thumb: [Object],
      authors: [Array]
    }
  ]
}
No images found! Can't build the thumb list without images. If you don't need thumbs, set showThumbs={false} in the Carousel. Note that it's not possible to get images rendered inside custom components. More info at https://github.com/leandrowd/react-responsive-carousel/blob/master/TROUBLESHOOTING.md
GET / 200 - - 590.873 ms
exx id  { exhibitId: '34' }
TypeError: Cannot read properties of undefined (reading 'length')
    at Exhibits (/var/app/build/index.js:1670:23)
    at Fb (/var/app/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.min.js:65:44)
    at Ib (/var/app/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.min.js:67:254)
    at W (/var/app/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.min.js:73:89)
    at Ib (/var/app/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.min.js:67:482)
    at W (/var/app/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.min.js:73:89)
    at Ib (/var/app/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.min.js:71:304)
    at W (/var/app/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.min.js:73:89)
    at Ib (/var/app/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.min.js:67:482)
    at W (/var/app/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.min.js:73:89)
GET /exhibits/34 500 - - 247.630 ms
```

**Comment by mrharpo** [comment]
Closed with #101
